OrbitGl/GpuTrack.cpp

```diff
@@ -50,6 +63,16 @@ GpuTrack::GpuTrack(TimeGraph* time_graph, StringManager* string_manager, uint64_
                              TriangleToggle::InitialStateUpdate::kReplaceInitialState);
 }
 
+void GpuTrack::OnTimer(const orbit_client_protos::TimerInfo& timer_info) {
+  // In case of having command buffer timers, we need to double the depth of the Gpu timers (as we
```

Comment:
Gpu -> GPU for consistency with other comments

Reply:
Done.